lisp/shell.lisp

```diff
@@ -3,67 +3,136 @@
 
 ;;; Macros to make working with the shell easier.
 
-;; Create an alias, intended to be used with executables not lisp code (use defn for that).
-(defmacro alias (name body) (progn
+(defmacro alias
+"
+Usage: (alias name body) or (alias name docstring body).
+
+Create an alias, intended to be used with executables not lisp code (use defn
+for that).
+
+Section: shell
+"
+	(name &rest args) (progn
+	(defq usage "Usage: (alias ll (ls -haltr)) or (alias ll \"ls show all files in reverse chronological order abd display size of each file..\" (ls -haltr)).")
 	(shell::register-alias name)
```

Comment:
i changed alias to allow for docstrings, its working for my aliases w and w/o docstrings properly, make sure it is with yours too to be sure?